### DISCUSSION TITLE
Getting an error with mini-starter dashboard config when using the browse entry -

### DISCUSSION THREAD

**Discussion by keithlxs** [nvim-mini/mini.nvim#610] [Q&A] [answered]
Hello, 

I just started using mini-starter and I like it so far. But I'm getting an error (screenshot attached)
 when I use the dashboard config that's in the documentation. Just the first option, 'Browse' is not working. All the other options open Telescope just fine. I'm not sure if there's a dependency I could be missing or if this is a bug. Here's the error when I use the 'Browse' option. I also have the Telescope plugin installed. 

E5108: Error executing lua vim/_editor.lua:0: nvim_exec2(): Vim:[telescope.run_command]: Unknown command
stack traceback:
        [C]: in function 'nvim_exec2'
        vim/_editor.lua: in function 'cmd'
        ...y/.config/nvim/plugged/mini.starter/lua/mini/starter.lua:1482: in function 'eval_fun_or_string'
        ...y/.config/nvim/plugged/mini.starter/lua/mini/starter.lua:874: in function 'eval_current_item'
        [string ":lua"]:1: in main chunk

Any help would greatly appreciated,  Thanks!

![Screenshot from 2023-12-10 21-09-10](https://github.com/echasnovski/mini.nvim/assets/1668110/a0b362a6-2a3d-49e5-b05b-8c9b444bf7f8)

**Comment by echasnovski** [comment, marked as answer]
Woah, thanks for this! It is indeed a semi-issue in 'mini.starter', as this item uses `:Telescope file_browser` command which is not a built-in [for already about two years now](https://github.com/nvim-telescope/telescope.nvim/pull/1453). It was moved to [nvim-telescope/telescope-file-browser.nvim](https://github.com/nvim-telescope/telescope-file-browser.nvim) which you'd have to install in order to use it.

I'll think about the best approach to resolve this in 'mini.starter' and update it (hopefully) soon. Besides, it might really need a similar new `sections.pick` which uses ['mini.pick'](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-pick.md).

**Reply by echasnovski** [reply]
I came to a conclusion that the best approach here is to document that "Browser" item requires a said dependency. This should be already added on latest `main`.

There is now also a `sections.pick` for 'mini.pick' pickers.

Thanks again for raising this issue!

**Reply by keithlxs** [reply]
Just to confirm, the new dependency is mini-pick right?
Thanks!

**Reply by echasnovski** [reply]
For "Browser" entry in "Telescope" section it is 'nvim-telescope/telescope-file-browser.nvim'.
